packages/runtime/test/src/interpreter/opcode-list.ts

```diff
@@ -2306,8 +2306,10 @@ describe("Teal Opcodes", function () {
         let op = new AppLocalGetEx([], 1, interpreter);
         op.execute(stack);
 
-        let flag = stack.pop();
-        assert.equal(BigInt('0'), flag);
+        let didExistFlag = stack.pop();
+        let val = stack.pop();
+        assert.equal(BigInt('0'), didExistFlag);
+        assert.equal(BigInt('0'), val);
```

Comment:
you can write `0n` instead of `BigInt('0')`. Also, for small numbers, we don't need to pass string into `BigInt`

Reply:
Yes, actually we are updating all `BigInt('X')` => `Xn` in [this](https://github.com/scale-it/algorand-builder/pull/184) PR.
Will update this one for now.